liqd/adhocracy3.mercator
# remove needless test fixtures, update pep257/pep8

- rm needless splinter (test browser) test fixtures
- update pytest (new option to run only failed tests)
- update pep8/pep257 (news: no new line above class docstring, no sourrounding spaces in docstring)
- run autopep8 --aggressive for all files
- check pep8/pep257 for all files
